src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1556,17 +1626,15 @@ void Convolution::initTryBrgconvFlag() {
             // heuristics: if it's int8 model and it has binary post ops we will not use brgconv
```

Comment:
I guess the comments should be updated.
Could you please also check all the corresponding comments in terms of relevance.

Reply:
Updated. Thx!